constructor/header.sh

```diff
@@ -349,6 +349,15 @@ if ! mkdir -p "$PREFIX"; then
     exit 1
 fi
 
+total_installation_size_kb="__TOTAL_INSTALLATION_SIZE_KB__"
+free_disk_space_bytes="$(df -Pk "$PREFIX" | tail -n 1 | awk '{print $4}')"
```

Comment:
Is `df` available universally on posix systems?

Reply:
[The wiki page](https://en.wikipedia.org/wiki/Df_(Unix)) reads like it's part of the POSIX spec, so I assumed so, but we can check if it's available before running this?

Reply:
Since it's part of POSIX spec, it's fine to not check.